<?xml version="1.0"?>

<launch>
	<arg name="log_path" />
	<arg name="frontier_selection" />

	<include file="$(find explorer)/launch/simple_navigation_pioneer.launch">
		<arg name="frontier_selection" value="$(arg frontier_selection)" />
		<arg name="log_path" value="$(arg log_path)" />
	</include>
        <rosparam file="$(find explorer)/param/pioneer3dx_energy_params.yaml" command="load"/>
</launch>
